etc/README.rst

```diff
@@ -4,27 +4,67 @@ Ceagle Portal Configuration
 Placement
 ---------
 
-config.json should be placed to /etc/ceagle/config.json
+Configuration file should be placed to */etc/ceagle/config.json*.
 
+If there is an environment variable *CEAGLE_CONF* set, it is used as priority.
+In this case it is important to use absolute path to configuration file.
+
+Example:
+
+.. code-block::
+
+    export CEAGLE_CONF=/home/username/configs/custom_config.json
+    ./runserver.sh
 
 Configuration File Description
 ------------------------------
 
-Configuration file is plain json document with 3 top level keys:
+Configuration file is plain JSON document with top level keys
+that describes Flask and available microservices configuration:
+
+Here is a simple example with configuration for Flask and two microservices:
+
+.. code-block::
+
+  {
+    "flask": {
+        "PORT": 5001,
+        "HOST": "0.0.0.0",
+        "DEBUG": true
+    },
+    "cloud_status": {
+        "enabled":  true,
+        "endpoint": "http://dummy.example.org/api/cloud_status"
+    },
+    "capacity": {
+        "enabled":  true,
+        "endpoint": "http://dummy.example.org/api/capacity"
+    }
+  }
+
+Flask configuration
+~~~~~~~~~~~~~~~~~~~
+
+Flask configuration is set via *flask* key and described in `official documentation <http://flask.pocoo.org/docs/0.11/config/>`_.
+
+The only extra options are HOST and PORT.
+
 
-* flask - Flask configuration arguments with Host and Port options added.
-* global - All global settings (e.g. cloud portal name)
-* cloud_status - cloud_status pages configuration
+Microservices configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+The only mandatory parameters are *enabled* and *endpoint*.
 
-Configuration Of cloud_status In Nuts
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+There are no limitations for other parameters.
 
-* enabled - If True initialize /cloud_status/ API and "Cloud status" menu
-* enabled_dashboards - initializes sub API /cloud_status/availability and /cloud_status/health with sub menus
-* regions - list of regions that should be displayed
-** region - displayed name of region
-** health - dict with 2 items connection URL to Elastic Search &  index from what to query health data production by
-            https://github.com/ceagle/health
-** availability - dict with single argument "graphite" that is used to specify connection string to graphite that stores metrics.
+Explanations for some parameters:
 
+* enabled - boolean, whetjer microservice is available
+* endpoint - microservice connection URL
+* regions - list of available regions
+* regions[]/region - region name
+* regions[]/health - health monitoring configuration
```

Comment:
So nope, there is no "regions" in this case.

All regions are maintained by 1 service not N services, so there won't be any information about regions in portal (it's returned from specific service)